<!doctype html>
<html lang="en">
<head>
    <meta charset="utf-8"/>
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    <meta http-equiv="content-type" content="text/html; charset=utf-8"/>
    <meta http-equiv="cleartype" content="on">
    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="400">
    <meta name="theme-color" content="#fff">
    <meta name="apple-mobile-web-app-status-bar-style" content="#fff">

    <meta name="description" content="uLoop.org - a virtual repository for uLoop and Loop assembly standardised parts.">
    <meta name="keywords" content="uLoop, DNA repository, standardised DNA assembly, plasmid construction, modular cloning, universal Loop assembly">
    <meta name="author" content="Bernardo Pollak">
    <meta name="robots" content="index, follow">
    <title>uLoop: universal Loop assembly</title>
    
    <link rel="icon" type="image/png" href="images/favicon.png">
    <link rel="stylesheet" href="css/normalize.css">
    <link rel="stylesheet" href="css/skeleton.css">
    <link rel="stylesheet" href="css/extras.css">
    <link rel="stylesheet" href="css/custom.css">
</head>
<body>

<div class="container">
        <a href="/#"><img style="width:200px;" src="img/uLoop.svg" alt="uLoop"></a>
        <div class="navbar">
    
            <a href="#mobile-menu2" class="toggle-visible right visible-phone menu-icon"></a>

            <ul class="left menu">
            <li><a class="item" href="domestication.html">Domestication</a></li>
            <!--
            <li><a class="item" href="http://loopdesigner.herokuapp.com" target="_blank">LoopDesigner</a></li>
            -->
            </ul>

            <!-- Not currently available>
            <ul class="right menu hidden-phone" id="mobile-menu2">
            <li><a class="item" href="#">Home</a></li>
            <li><a class="item" href="#">Settings</a></li>
            <li class="hover dropdown">
            <span class="item">Dropdown on hover<i class="caret"></i></span>

            <ul>
              <li><a class="item" href="#">Foo</a></li>
              <li><a class="item" href="#">Bar</a></li>
              <li><a class="item" href="#">Baz</a></li>
            </ul>
            </li>
            <li><a class="item" href="#">Logout</a></li>
            </ul>
            <Close commenting-->


        </div>  <!-- Navbar /div -->
        <div style="margin-top: 10px">
            <h3>Domestication</h3>
            <p>
                This refers to the process of making DNA sequences usable with Type IIS assembly systems. The requirements are: <br />
                (<b>1</b>) that the sequence does not contain sites for the Type IIS restriction enzymes used in the assembly system, since this would interfere with the reaction. For Loop and uLoop. <b>BsaI</b> and <b>SapI</b> sites.<br />
                (<b>2</b>), that flanking overhangs are added to the sequence to make it a <i>standard part</i>. These flanking overhangs have been predefined to provide further functionality.

                <h3>Syntax</h3>
                <a href="https://nph.onlinelibrary.wiley.com/doi/full/10.1111/nph.15625" target="_blank" alt="Loop paper">Loop</a> & uLoop follow the <a href="https://doi.org/10.1111/nph.13532" alt="Common syntax" target="_blank">common syntax</a> for standardised DNA components used in Type IIS assembly systems proposed in the plant field. This ensures that standard parts are interoperable with other Type IIS assembly systems as <a href="https://www.addgene.org/cloning/moclo/marillonnet/?gclid=CjwKCAiAhp_jBRAxEiwAXbniXSm8MOCFUcFgq1Uw9PQxTYjAQIYC1ExxYce4lVlabUnClH1A-fW5fRoCOIEQAvD_BwE" target="_blank" alt="MoClo">MoClo</a> and <a href="https://gbcloning.upv.es/" target="_blank" alt="GoldenBraid">GoldenBraid</a>, given that no <b>SapI</b> sites are present.<br />
                The "common syntax" defines 12 overhangs for flanking subgenetic DNA sequences in accordance to a prototypical eukaryotic transcription unit.
            </p>
            <a href=""><img class="center-img" style="width: 550px" src="img/common-syntax.png" alt="Common Syntax"></a> 
            <p>
                <br />
                For uLoop we will be using the following specification according to the common syntax:
            </p>
            <a href=""><img class="center-img" style="width: 550px" src="img/uLoop-sintax.svg" alt="Common Syntax"></a> 

            <h3>Recode2S</h3>
            <p>
                To help with the domestication of sequences, we have developed <a href="http://recode2s.herokuapp.com" target="_blank" alt="Recode2S">Recode2S</a>, an application to simplify searching for restriction sites and adding overhangs. Recode2S is also frame-aware, in order to avoid changing codons by selecting silent mutations if the site is not in the reading-frame of the CDS.
                <br />
                <b>Important</b> The output of Recode2S shows how your sequence should look like once domesticated as a L0 standard part. From this output, we will use the region between (and including) the flanking overhangs without the BsaI sites.
                <br />
                If the sequence does not contain internal BsaI and SapI sites continue to the next section. Otherwise, read through the rest of the document and go to the last section.
            </p>
            <h3>Domestication vector pL0R-lacZ</h3>
            <p>We have created a entry vector that enables domestication through overlap-dependent assembly as well as SapI-mediated restriction ligation. The pL0R-lacZ vector contains a lacZ negative selection marker, a spectynomycin resistance cassette and restriction sites for BsaI and SapI. SapI restriction releases lacZ for entry of domesticated sequences while at the same time reconstituting BsaI sites. Then, cloned sequences can be used in a L0 to L1 assembly. The vector contains unique nucleotide sequences (UNS) for sequencing and cPCR verification.</p>

            <a href=""><img class="center-img" style="width: 400px" src="img/pL0R.svg" alt="Domestication vector"></a> 
            <p>To domesticate DNA sequences through SapI-mediated restriction/ligation, flanking sequences have to included in the DNA insert in order to enable SapI restriction, overhangs for reconstitution of the BsaI site in the L0 receiver and overhangs according to the common syntax. A specific set of 3 bp overhangs exposed by SapI digestion been designed for efficient assembly into the L0 receiver which contain distinct 5’ and 3’ overhangs. These 3 bp overhangs will reconstitute the BsaI site that are exposed in the receiver when the L0 receiver is digested by SapI along with the N nucleotide spacer between the BsaI restriction site and the common syntax overhangs.<br /> <br /> 
            SapI is shown in boldface, 3 bp overhangs are underlined, NNNN corresponds to the 4 bp common syntax overhangs and XXX corresponds to the DNA module.
            </p>
            <a href=""><img class="center-img" style="width: 500px" src="img/pL0R-mechanism.svg" alt="Domestication mechanism"></a> <br />
            <p> If your sequence does not require removing BsaI and SapI sites, then add <b>AAGCTCTTCATCC</b> at the 5' end, and <b>CGAAGAAGAGCAA</b> at the 3' end through PCR.
            <a href=""><img class="center-img" style="width: 500px" src="img/Domesticate.svg" alt="Domesticate design"></a>

            <h3>Domesticating internal BsaI and SapI sites</h3>
            <p>
            Sites must be removed through mutation (PCR) or by synthesis (gBlocks). If few sites are present (2 or less), then PCR might be a suitable way of removing these sites unless the sequence contains repetitive sequences or  low-complexity regions. <br /><br />
            For each site present, the sequence will be split in that section to change a single nucleotide. Several techniques can be used to eliminate the sites (overlap-extension PCR, Gibson assembly, homologous recombination), but in this case we will use SapI-mediated directional assembly. To achieve this, every section will include flanking SapI sites included by PCR as in the previous section, however the main challenge will be to find <i>unique</i> overhangs that will enable the reaction to assemble the sections in the correct order. Since we will be using SapI, overhangs are 3 bp and they cannot be too similar (hopefully 2 or more dissimilar bases), be of low-complexity or either TCC, CGA, TCG or GGA. It does not matter, however, if the mutated restriction site base pair is any position of the overhang.
            </p>
            <a href=""><img class="center-img" style="width: 500px" src="img/Domesticate-multiple.svg" alt="Domesticate multiple"></a>
            In this case, the primers used to create the SapI sites should cut directly in the selected site. This is accomplished by adding <b>AAGCTCTTCA</b> to the 5' of the oligonucleotide (blue). The flanking oligonucleotides (black) still contain same sequence as described previously.
             <br /><br /> <br /><br />
        </div>

    </div> <!-- Container /div -->
</body>
</html>